pkgs/development/python-modules/unidiff/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "unidiff";
+  version = "0.5.5";
+
+  src = fetchFromGitHub {
```

Comment:
I'm seeing this distributed on PyPi https://pypi.org/project/unidiff/

Reply:
it's because the PyPI tarball doesn't have tests, I've added a comment